docs/guide/multisite.md

```diff
@@ -2,109 +2,189 @@
-        second_site:
-            pagelayout: second_site/pagelayout.html.twig
-            content_view:
-                full:
-                    blog_post:
-                        template: "second_site/full/blog_post.html.twig"
-                        match:
-                            Identifier\ContentType: [blog_post]
+        kids_group:
+            content:
+                tree_root:
+                    # LocationId of "Children"
+                    location_id: 57
 ```
 
-This config defines views that will be used for the `second_site` siteaccess: a separate `second_site/pagelayout.html.twig` and a template to be used for blog posts. When no view is defined under `second_site`, such as in the case of the `blog` Content Type, the template defined under `default` will apply. `default` will also be used for all siteaccesses other than `second_site`.
+Thanks to this config, you can access `<your site>/kids/Articles/Lions`, but not `<your site>/kids/General/Articles/Mammals`.
+
+In the next step, you need to reuse some content between sites, for example "Animal-photos" from "Media".
+You can allow the children's site to access them, even though they are in a different part of the tree, via `excluded_uri_prefixes`:
+
+``` yaml
+        kids_group:
+            content:
+                tree_root:
+                    location_id: 57
+                    excluded_uri_prefixes: [ /media/images/animal-photos ]
+```
 
-To load the base layout in templates you now need to use `{% extends noLayout == true ? viewbaseLayout : pagelayout %}`.
+At this point, using the `kids` SiteAccess, you gain access to `<your site>/kids/Media/Images/Animal-photos`,
+despite the fact that it is not a sub-item of the "Children" Location.
 
 ### Setting the Index Page
 
-The Index Page is the page shown when the root index / is accessed.
-You can configure the Index Page separately for each siteaccess. Put the parameter `index_page` in your `ezplatform.yml` file, under the right siteaccess category.
+The Index Page is the page shown when the root index `/` is accessed.
+You can configure the Index Page separately for each SiteAccess. Place the parameter `index_page` in your `ezplatform.yml` file, under the correct SiteAccess:
 
 ``` yaml
 # ezplatform.yml
 
 ezpublish:
     system:
-        mygreat_site:
-            languages:
-                - eng-US
-            #The page to show when accessing IndexPage (/)
-            index_page: /yourURIPage
+        kids_group:
+            # The page to show when accessing IndexPage (/)
+            index_page: /KidsFrontPage
 ```
 
 If not specified, the `index_page` is the configured content root.
+
+### Multisite with different repositories
+
+!!! caution
+
+    Only one repository (database) can be used per domain.
+    This does not prohibit using subdomains, but they must all use the same repository, for example:
+
+    - `ez.no` domain can use `ez.repo`
```

Comment:
Now that I see this, instead of `.` in the repo names here which can be confused for levels in Symfony config tree, lets rather use `_`. So here `ez_repo` and on `doc_repo` instead of `doc.repo`.